elastic/kibana
# [Security Solution][Alert Details] - remove flyout tour introduced in 8.14

## Summary

We [added](https://github.com/elastic/kibana/pull/180318) a guided tour to the expandable flyout back in `8.14`. It is time to remove it as enough users have seen it.

No UI changes other than the tour not showing up.

### How to test

- clear local storage (more specifically remove the `securitySolution.documentDetails.newFeaturesTour.v8.14` key
- open a alert or event details flyout and verify that the tour does not show up

### Checklist


Should help solve https://github.com/elastic/kibana/issues/197492
